### ISSUE TITLE
Support for download explode

### ISSUE THREAD

**Issue by MichaelSiggelkowRnS** [JFrogDev/jfrog-cli-go#111]
Similar to the build info version of spec files (https://www.jfrog.com/confluence/display/RTF/Using+File+Specs#UsingFileSpecs-DownloadSpecSchema) I would like to request the support for "explode" parameter also for the CLI version.

The overall rational is to have one spec file and allow the usage of the CLI tool for developer environment and the build info version in a Jenkins environment. thanks!

**Comment by eyalbe4** [comment]
@MichaelSiggelkowRnS,
Version 1.14.0 has bee released and includes a new *--explode* option for the *rt download* command.
We'd appreciate your feedback for the new functionality.

**Comment by Dineshk77** [comment]
Hai Just want to check that this --explode option will also preserve the symlink on downloading a zip from artifactory?.

**Comment by eyalbe4** [comment]
@Dineshk77,
The explode option does nothing more than extracting an archive when it is download. The exploded archive structure and content should therefore be preserved.

**Comment by eyalbe4** [comment]
@Dineshk77,
Please let us know if you see that the symlinks are not preserved or if you need further assistance.

**Comment by Dineshk77** [comment]
@eyalbe4 Yes the symlinks are not preserved. I am trying to download a .zip from artifactory using the latest jfrog cli.
command used - `jfrog rt dl --flat=true --explode=true --threads=30 source target`
Package is downloaded and extracted perfectly to the mentioned target. However the symlinks weren't preserved. Please let me know if any other option has to be used. thanks.

OS - MAC.

**Comment by eyalbe4** [comment]
@Dineshk77,
How was the zip created initially?

**Comment by Dineshk77** [comment]
@eyalbe4 Zip file was created on the linux machine using the `zip` command. when I unzip using `unzip` command all the symlinks are preserved.

**Comment by mafahir** [comment]
@eyalbe4 i am using jfrog 16.2, i have a zip with symlinks inside.
when i use the explode option symlinks are broken. if i unzip manually they work fine.
i tried using option "symlinks":"true" and  "validate-symlinks": "true" doesnt help.

**Comment by ted-poole** [comment]
I have also just discovered that downloading a tgz archive with explode=true _doesn't_ preserve symlinks.
If I download it with explode=false and then extract the archive with tar it _does_ preserve symlinks.

**Comment by chgans** [comment]
I didn't have issue with symlinks using jfrog CLI on Linux, however using `rtDownload` [1] I do have the problem.
On my computer:
```
$ ls -l external-dependencies/boost/armeabi-v7a/include/boost
lrwxrwxrwx 1 chgans chgans 16 Jun  5 19:30 external-dependencies/boost/armeabi-v7a/include/boost -> boost-1_69/boost
$ ls -l external-dependencies/boost/armeabi-v7a/include/boost/ | head
total 1676
drwxr-xr-x  5 chgans chgans  4096 Jun  5 19:30 accumulators
drwxr-xr-x  7 chgans chgans  4096 Jun  5 19:30 algorithm
drwxr-xr-x  3 chgans chgans  4096 Jun  5 19:30 align
-rw-r--r--  1 chgans chgans   653 Jun  5 19:30 aligned_storage.hpp
-rw-r--r--  1 chgans chgans   633 Jun  5 19:30 align.hpp
-rw-r--r--  1 chgans chgans  9302 Jun  5 19:30 any.hpp
drwxr-xr-x  5 chgans chgans  4096 Jun  5 19:30 archive
-rw-r--r--  1 chgans chgans 15903 Jun  5 19:30 array.hpp
drwxr-xr-x 12 chgans chgans  4096 Jun  5 19:30 asio
```

From the pipeline:
```
20:00:58  + ls -l external-dependencies/boost/armeabi-v7a/include/boost
20:00:58  -rw-r--r-- 1 root root 0 Aug 11  2020 external-dependencies/boost/armeabi-v7a/include/boost
20:00:58  + ls -l external-dependencies/boost/armeabi-v7a/include/boost/
20:00:58  ls: cannot access 'external-dependencies/boost/armeabi-v7a/include/boost/': Not a directory
```


[1] https://www.jfrog.com/confluence/display/JFROG/Declarative+Pipeline+Syntax#DeclarativePipelineSyntax-UploadingandDownloadingFiles

**Comment by shuvadipc** [comment]
Outdated version.
